src/test/java/uk/ac/ebi/eva/pipeline/configuration/validation/VepCachePathValidatorTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.configuration.validation;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.batch.core.JobParametersBuilder;
+import org.springframework.batch.core.JobParametersInvalidException;
+
+import uk.ac.ebi.eva.pipeline.configuration.JobParametersNames;
+import uk.ac.ebi.eva.pipeline.configuration.validation.step.VepInputGeneratorStepParametersValidatorTest;
+
+public class VepCachePathValidatorTest {
```

Comment:
Other tests:
* The path could point to a file instead of a directory
* The path could be not readable